vitrina/datasets/services.py

```diff
@@ -105,3 +106,7 @@ def can_create_dataset(user: User, org_id) -> bool:
         if user.is_staff:
             permission = True
     return permission
+
+
+def can_see_dataset_members(user: User, dataset: Dataset) -> bool:
+    return Representative.objects.filter(organization_id=dataset.organization_id, email=user.email).exists()
```

Comment:
`user.is_staff` and `user.is_superuser` should have access to this. Please collaborate with the #171.